tests/erc721/erc721_test.go

```diff
@@ -543,6 +545,81 @@ func TestOpenSeaProxyPreApprovalOptInOut(t *testing.T) {
 	}
 }
 
+func TestRoyalties(t *testing.T) {
+	sim, nft, _ := deploy(t)
+
+	tests := []struct {
+		setConfig    bool
+		newReceiver  common.Address
+		newPermyriad int64
+		salesPrice   int64
+	}{
+		{
+			salesPrice: 200000,
+		},
+		{
+			setConfig:    true,
+			newReceiver:  sim.Addr(deployer),
+			newPermyriad: 300,
+			salesPrice:   500000,
+		},
+	}
+
+	for _, tt := range tests {
+		if tt.setConfig {
+			nft.SetDefaultRoyalty(sim.Acc(deployer), tt.newReceiver, big.NewInt(tt.newPermyriad))
+		}
+
+		wantAmount := tt.salesPrice / 1000 * 75
+		wantReceiver := sim.Addr(royaltyReceiver)
+
+		if tt.setConfig {
+			wantAmount = tt.salesPrice / 10000 * tt.newPermyriad
+			wantReceiver = tt.newReceiver
+		}
+
+		receiver, amount, err := nft.RoyaltyInfo(nil, big.NewInt(0), big.NewInt(tt.salesPrice))
+
+		if err != nil || big.NewInt(wantAmount).Cmp(amount) != 0 || wantReceiver != receiver {
+			t.Errorf("RoyaltyInfo(0, %d) got (%s, %d), err = %v; want (%s, %d), nil err", tt.salesPrice, receiver, amount, err, wantReceiver, wantAmount)
+		}
+
+	}
+}
+
+func TestInterfaceSupport(t *testing.T) {
+	_, nft, _ := deploy(t)
+
+	tests := []struct {
+		interfaceId string
```

Comment:
`interfaceID` even though it's different in Solidity. 

https://github.com/golang/go/wiki/CodeReviewComments#initialisms